tests/tests_catapult_integration.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+ROOT_DIR="$(git rev-parse --show-toplevel)"
+export CLUSTER_NAME=test
+export SCF_OPERATOR=true
+export BACKEND=kind
+export ENABLE_EIRINI=false
+export CHART_URL=https://scf-v3.s3.amazonaws.com/scf-3.0.0-380c355.tgz
```

Comment:
Would it be possibleto  use a symlink to a working scf v3 on s3 to not hardcode a chart?

Reply:
It exhaust the travis limits anyway, so we can't do this :(